README.md

```diff
@@ -163,10 +165,11 @@ Medical Robotics Junior Faculty Forum (ISMR: <a href="https://junior-forum-ismr.
 
 ## Table Of Contents
 
-  * [User community](#user-community)
```

Comment:
Is there any reason why the table of contents is changed?

Reply:
These modifications were accidental and, actually, I don't know how it happened... sorry!